examples/nextjs-trpc/package.json

```
@@ -0,0 +1,46 @@
+{
+	"name": "nextjs-trpc",
+	"version": "0.1.0",
+	"private": true,
+	"type": "module",
+	"scripts": {
+		"build": "next build",
+		"check": "next lint && tsc --noEmit",
+		"dev": "next dev --turbo",
```

```suggestion
		"dev": "next dev",
```

I wasn't able to get this running unless I removed the `--turbo` flag, it was failing to import `@arcjet/next`. It seems that relative imports are known to be broken when using next dev with turbo. 

https://github.com/vercel/next.js/issues/69426

I don't think the solution they propose will work either because its for importing `.ts` files, but its probably fine to just not use turbo here. Curious to hear if @blaine-arcjet has any better solutions though.

---

Removed the flag for now.